chia/data_layer/data_layer.py

```diff
@@ -260,9 +260,10 @@ async def fetch_and_validate(self, subscription: Subscription) -> None:
             await self.data_store.create_tree(tree_id=tree_id)
 
         while True:
-            server_info = await self.data_store.maybe_get_server_for_store(tree_id)
+            timestamp = int(time.time())
```

Comment:
is it safe to use `time.monotonic()` here? According to the documentation: `The reference point of the returned value is undefined, so that only the difference between the results of two calls is valid.` so I sticked with `time.time()` for now.